### DISCUSSION TITLE
Exception while painting

### DISCUSSION THREAD

**Discussion by Goldenia** [ho-dev/HattrickOrganizer#1190] [Q&A]
Hi,
just down portable version HO-4.1.3310.2-portable-win-JRE.zip.
Install is ok but launching product loop with following error :

"java.lang.RuntimeException: Exception while painting 'jar:file........."

When you check error messages, ui seems to down some part till next "same" error.

OS : Windows 8.1 64bits

Thanks for help. Tell if more info needed.

Same with HO-5.0.3778.0-portable-win-DEV-JRE.zip

**Comment by akasolace** [comment]
@Goldenia are you also experiencing this issue with latest release?
How are you launching the app?
